config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
-  
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
-  root to: "texts#index"
-
+  root "movies#index"
```

Comment:
トップページはどちらでもよいですが，元アプリはテキスト教材がトップページですね。
```suggestion
  root "texts#index"
```